Util/openl-opentelemetry-javaagent-extension/src/org/openl/opentelemetry/javaagent/extension/OpenLMethodsInstrumentationModule.java

```diff
@@ -0,0 +1,35 @@
+package org.openl.opentelemetry.javaagent.extension;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+
+import java.util.List;
+import java.util.Set;
+
+@AutoService(InstrumentationModule.class)
+public class OpenLMethodsInstrumentationModule extends InstrumentationModule {
+
+    public OpenLMethodsInstrumentationModule() {
+        super("openl-methods");
+    }
+
+    @Override
+    public List<String> getAdditionalHelperClassNames() {
+        return List.of(MethodSingletons.class.getName());
+    }
+
+    @Override
+    public List<TypeInstrumentation> typeInstrumentations() {
+        return List.of(
+                new ExecutableRulesMethodSpanGenerator(
+                        Set.of(
+                        "org.openl.rules.dt.DecisionTable",
+                        "org.openl.rules.calc.Spreadsheet",
+                        "org.openl.rules.tbasic.Algorithm",
+                        "org.openl.rules.cmatch.ColumnMatch",
+                        "org.openl.rules.method.table.TableMethod"))
```

Comment:
What is the reason to define constant values there? Can be they moved inside `ExecutableRulesMethodSpanGenerator`? Is `ExecutableRulesMethodSpanGenerator` reusable ?